examples/ui.frontend/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "aem-maven-archetype",
+  "version": "1.0.0",
+  "description": "Generates an AEM Frontend project with Webpack",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/aem-project-archetype"
+  },
+  "private": true,
+  "main": "src/main/webpack/site/main.ts",
+  "license": "SEE LICENSE IN LICENSE.txt",
+  "scripts": {
+    "dev": "webpack -d --env dev --config ./webpack.dev.js && clientlib --verbose",
+    "prod": "webpack -p --config ./webpack.prod.js && clientlib --verbose",
+    "start": "webpack-dev-server --open --config ./webpack.dev.js",
+    "sync": "aemsync -d -p ../ui.apps/src/main/content",
+    "watch": "webpack-dev-server --config ./webpack.dev.js --env.writeToDisk & watch 'clientlib' ./dist & aemsync -w ../ui.apps/src/main/content"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.3.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.3.2",
+    "@babel/preset-react": "^7.10.4",
+    "@babel/preset-typescript": "^7.10.4",
+    "@typescript-eslint/eslint-plugin": "^2.14.0",
+    "@typescript-eslint/parser": "^2.14.0",
+    "acorn": "^6.1.0",
+    "aem-clientlib-generator": "^1.4.3",
+    "aemsync": "^4.0.1",
+    "autoprefixer": "^9.2.1",
+    "babel-loader": "^8.1.0",
+    "browserslist": "^4.2.1",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^5.0.4",
+    "css-loader": "^3.0.0",
+    "cssnano": "^4.1.10",
+    "eslint": "^6.8.0",
+    "eslint-loader": "^3.0.3",
+    "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.4.4",
+    "node-sass": "^4.11.0",
+    "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "postcss-loader": "^3.0.0",
+    "sass": "^1.17.2",
+    "sass-loader": "^7.1.0",
+    "source-map-loader": "^0.2.4",
+    "style-loader": "^0.14.1",
+    "terser-webpack-plugin": "^1.4.1",
+    "ts-loader": "^5.3.3",
+    "tsconfig-paths-webpack-plugin": "^3.2.0",
+    "typescript": "^3.3.3333",
+    "watch": "^1.0.2",
+    "webpack": "^4.27.1",
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.9.0",
+    "webpack-import-glob-loader": "^1.6.3",
+    "webpack-merge": "^4.2.1"
+  },
+  "dependencies": {
+    "@adobe/aem-core-cif-react-components": "^1.4.0",
```

Comment:
Might not be that important, since we only use this for one React component. The examples project uses the same version of dependencies as built (including snapshots), while the React components dependency points to the release. In the build of `react-components`, we already do an `npm link`, so maybe we could add an `npm link @adobe/aem-core-cif-react-components` to the build here as well to make sure, we use the exact same version as in the components build?

Reply:
Good point, I actually tried last week that but it didn't work ... the build then failed with some weird webpack error. I can create a follow-up error for that, that PR is long enough. ;-)